### DISCUSSION TITLE
General question for pimcore | It's About Variant

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#12046] [General]
Hello,

Pimcore provides the variant functionality, so in the demo site, we have variants of the Car so why does Pimcore use the childe object instead of the variant?

Is there any limitation for use variants or anything else?

There is no blocker for me but I am just thinking about this.

Thanks.

**Comment by kingjia90** [comment]
maybe this could be a limitation, currently [search & move doesn't support Variants](https://github.com/pimcore/pimcore/issues/11854) yet.
